### DISCUSSION TITLE
Question : Can issues from a bridge embedded in the git repo ?

### DISCUSSION THREAD

**Discussion by nicobo** [git-bug/git-bug#1193] [Q&A] [answered]
I can't figure this from the docs : are issues pulled from github (or another bridge) saved inside the git repo and can they be pushed to a remote for use without github.

My use case is :
1. fetch issues from github (or any other supported bridge)
1. **save** issues in the git repo so that it acts as a backup storage
1. when the time has come, leave github and only use git-bug and the locally saved issues

I could create the bridge, fetch issues, list them.
When doing `git bug push` nothing appears in the remote (github).
When doing `git bug bridge push` github issues are correctly updated but still no change can be observed in the remote git repo itself.
So it seems that bridges are a totally separate workflow from the native workflow and that I can't migrate from github to native ?

**Comment by sudoforge** [comment, marked as answer]
`git-bug-push` pushes bugs to the `refs/bugs` namespace. GitHub **only** shows the `refs/heads` namespace in their UI; they offer no way to view any other namespace. Thus, unfortunately, you will never be able to observe bugs pushed this way.

`git-bug-bridge-push` will push issues to the bridge, in your case this means that the bugs are "pushed" to GitHub Issues via the GitHub Issues API.

Let me know if you need any additional clarification!
